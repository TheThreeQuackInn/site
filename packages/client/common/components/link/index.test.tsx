import React from 'react';
import {create} from 'react-test-renderer';
import Link from './';

describe('Link', () => {
    it('renders correctly', () => {
        const component = create(<Link label="Go to" href="#" target="_blank" />);
        expect(component).toMatchSnapshot();
    });
});
