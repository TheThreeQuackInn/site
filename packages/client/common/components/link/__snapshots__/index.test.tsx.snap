// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<a
  className="transition-colors duration-300 mx-1 font-medium hover:underline link focus:outline-black dark:focus:outline-white"
  href="#"
  target="_blank"
>
  Go to
</a>
`;
